src/engraving/libmscore/edit.cpp

```diff
@@ -4856,7 +4856,7 @@ static EngravingItem* findLinkedVoiceElement(EngravingItem* e, Staff* nstaff)
     Measure* measure = segment->measure();
     Measure* m       = score->tick2measure(measure->tick());
     Segment* s       = m->findSegment(segment->segmentType(), segment->tick());
-    return s->element(dtrack);
```

Comment:
Why is it null?

Reply:
In the score in issue https://github.com/musescore/MuseScore/issues/14606, it's due to a different number of measures in the part vs. the masterscore, which I think probably shouldn't be possible? I've tried to recreate that score and I couldn't.

That makes me think the problem in https://github.com/musescore/MuseScore/issues/14200 is similar, though there are a lot more parts. I've asked for clarification from the submitter of that issue as to what steps were taken to see if I can narrow down a reason for that issue. I am running a debug build and #14200 is not crashing for me on master, but if a nullcheck here fixes it I can only assume there is a similar situation going on.

Reply:
The level above has a nullcheck though: https://github.com/musescore/MuseScore/blob/13f3d886c65017e591a9e69f0a10e4f43b4b6dd7/src/engraving/libmscore/edit.cpp#L4938

Which means that we already expect that `findLinkedVoiceElement` can return `nullptr`.